tests/yam/agama/boot_agama.pm

```diff
@@ -48,12 +49,18 @@ sub run {
     }
     my @params = split ' ', trim(get_var('EXTRABOOTPARAMS', ''));
 
-    $grub_menu->expect_is_shown();
-    $grub_menu->edit_current_entry();
-    $grub_entry_edition->move_cursor_to_end_of_kernel_line();
-    $grub_entry_edition->type(\@params);
-    $grub_entry_edition->boot();
-    $agama_up_an_running->expect_is_shown();
+
+    if (is_pvm_hmc()) {
```

Comment:
this should be up in line 40 with an `else` and finish with a `return` as early exist so nothing else is executed.

Reply:
817b033f2